pkgs/development/python-modules/pytest-flake8/default.nix

```diff
@@ -0,0 +1,25 @@
+{lib, buildPythonPackage, fetchPypi, pytest, flake8}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "pytest-flake8";
+  version = "0.8.1";
+
+  propagatedBuildInputs = [ pytest flake8 ];
```

Comment:
Why `pytest` shouldn't be added as `propagatedBuildInput`? It is a runtime dependency (see https://github.com/tholo/pytest-flake8/blob/master/pytest_flake8.py#L10) and also listed in `install_requires` (see https://github.com/tholo/pytest-flake8/blob/master/setup.py#L38).